src/resources/appsCampaignsResource.js

```diff
@@ -0,0 +1,12 @@
+const CampaignsResource = require('./campaignsResource');
```

Comment:
This file seems unnecessary, right? It is just a constructor with super. Maybe we can just use the `CampaignsResource` in `src/resources/appsProduct.js`. So for example

```js
this.campaigns = new CampaignsResource(baseUrl, signatureFactory, config);
```